spyder/widgets/panels/manager.py

```diff
@@ -94,7 +94,7 @@ def get(self, name_or_klass):
         :param name_or_klass: Name or class of the panel to retrieve.
         :return: The specified panel instance.
         """
-        if not isinstance(name_or_klass, str):
+        if not is_text_string(name_or_klass, str):
```

Comment:
@rlaverde, this call is wrong because `is_text_string` only receives one argument.